lbz/authz/wrappers.py

```diff
@@ -0,0 +1,64 @@
+# coding=utf-8
+"""
+Authorization module.
```

Comment:
You named this module `wrappers.py` so here, you should update that as well.

Reply:
But in general, I am not sure if this name is fully correct. Only one function here is a decorator. Maybe we should put this everything into the `permissions.py` module together with the `AuthzCollector` class 🤔

What do you think?

Reply:
Its to avoid import issues - circular imports. - fixed description

Reply:
Ok, I see it but still, the name `wrappers.py` is not fully precise 🤔